pywps/exceptions.py

```diff
@@ -34,7 +34,7 @@ class NoApplicableCode(HTTPException):
     Base exception class
     """
 
-    code = 400
+    code = 500
```

Comment:
The default `code` param below is 400, yet the above line is changed to 500.  Do these need to be consistent?